pkg/exporter/syslogexporter/README.md

```diff
@@ -1,29 +1,46 @@
 # Syslog Exporter
 
-Export/Forward syslog messages to a syslog server
-
 **Stability level**: Experimental
 
-This exporter can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/)
+## About The Exporter
+
+The syslog exporter/forwarder supports sending messages to a remote syslog server.
+
+- This exporter/forwarder can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/) using RFC5424 and RFC3164
+- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format
+- The syslog exporter must be used with the [syslog_parser](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/syslog_parser.md)
```

Comment:
Are we planning to document what the actual requirements are later, along with having configurable defaults for the missing fields?

Reply:
As of now, the `syslog_parser` is the only identified dependency/requirement that needs to be configured in the receiver. Which missing fields are you referring to?

Reply:
What I mean is that the exporter will work even if the syslog processor is not present, it'll simply use default values for properties. I think we should document this fact, rather than just saying it "must be used" with the processor.